bitcoin.html

```diff
@@ -480,6 +480,7 @@ <h3>High Quality Blogs:</h3>
             <li><a href="http://blog.oleganza.com/" target="_blank">Oleg Andreev</a></li>
             <li><a href="https://petertodd.org/" target="_blank">Peter Todd</a></li>
             <li><a href="https://gendal.me/tag/bitcoin/" target="_blank">Richard Gendal Brown</a></li>
+            <li><a href="<li><a href="http://startupmanagement.org/blog/" target="_blank">William Mougayar</a></li>
```

Comment:
You have a double `<li><a href="` in your commit. After `href="` should just come the URL